order/supplier/receive/README.md

```diff
@@ -70,16 +75,27 @@ The `buyerAccountNumber` should be set on forehand in the Tradecloud connection
 * `mergedItemDetails`: detailed part information provided by both buyer and supplier, see [Item details](./#item-details).
 * `lastUpdatedAt`: is the latest date time the order line has been changed, usefull for polling.
 
+### Current delivery schedule
+
+* `deliverySchedule`: the current aggregated delivery schedule with logistics info, see [Native Delivery Schedule](#native-delivery-schedule) below.
+* `deliveryScheduleIncludingRequests`: the current aggregated delivery schedule including any open supplier or buyer requestso, see [Native Delivery Schedule](#native-delivery-schedule) below.
+
 {% hint style="info" %}
 It is advised to use `deliverySchedule` with `prices` or alternatively `deliveryScheduleIncludingRequests` with `pricesIncludingRequests`.
 
-These fields give a summary of the current delivery schedule and prices. Use the `IncludingRequests` fields when you also send proposal or reopen requests events to your ERP.
+These fields give a summary of the current delivery schedule and prices. Use the `IncludingRequests` fields when you also send proposal or reopen request events to your ERP.
 
 When using these fields it is not necessary to use the `deliverySchedule` and `prices` fields in `buyerLine`, `buyerLine.requests`, `supplierLine.requests` or `confirmedLine`.
 
-`deliveryScheduleIncludingRequests`, `prices` and `pricesIncludingRequests` are only available in the new webhook, using the "Orders Webhook Integration" configuration in your company profile page, and also available in the `order-search` API when using polling.
+`deliveryScheduleIncludingRequests`, `prices` and `pricesIncludingRequests` are only available in the new webhook, using the "Orders Webhook Integration" configuration in your company profile page, and are also available in the `order-search` API when using polling.
 {% endhint %}
 
+## Current delivery line
+
+* `scheduledDelivery`: the current aggregated delivery line with logistics info, see [Simple Delivery Schedule](#simple-delivery-schedule) below.
+
+In case of the simple delivery schedule, there is no `scheduledDeliveryIncludingRequests` variant available. Please let [support](support.md) know when you need this field.
```

Comment:
Also, `scheduledDelivery` is actually based on `deliveryScheduleIncludingRequests`.
So it's `deliverySchedule` that is not present.

Reply:
It says `scheduledDeliveryIncludingRequests` eg. `scheduledDelivery` + `IncludingRequests` variant